### ISSUE TITLE
bug: gpg related errors in Ubuntu on Windows via WSL 1

### ISSUE THREAD

**Issue by patrickserrano** [automotiveMastermind/prompt#69]
After updating to prompt 8.0 I was getting the following when opening a new prompt:
```
prompt: using zsh shell
gpgconf: error running '/usr/bin/gpg-connect-agent': exit status 1
gpgconf: error running '/usr/bin/gpg-connect-agent NOP': General error
prompt: latest version already installed: be80c613ad66508acbc9061dd40bbbf3cac5ea8b.
patrick@VM : ~
```

Running `gpg` and `gpgconf` seems to have resolved the issue. But it's a pretty bad first run/upgrade experience for users who have't used gpg before. 

System Information:
- Prompt 8.0.0
- Windows 10 running Ubuntu 2004.2020.812.0 via WSL1

**Comment by dmccaffery** [comment]
Interesting. Must be specific to Ubuntu as configured in WSL1.

**Comment by jeffreypmoran** [comment]
Hi @dmccaffery - @patrickserrano sent me here after I mentioned that I ran into this issue on my machine after running update-prompt today.  The difference was that this wasn't in WSL1, it was in the terminal on my mac (Catalina, macOS 10.15.6).

```
prompt: using bash shell
gpgconf: error running '/usr/local/Cellar/gnupg/2.2.26/bin/gpg-connect-agent': exit status 1
gpgconf: error running '/usr/local/Cellar/gnupg/2.2.26/bin/gpg-connect-agent NOP': General error
prompt: a new version of prompt is available: a95f87e6bdba2404804fb3ba30396805c1b41102
  - run the update-prompt command line tool to upgrade
```
